src/modules/formBuilder/components/FormEditorItemProperties.tsx

```diff
@@ -0,0 +1,182 @@
+        />
+        <TextInput
+          label='Read-only label'
+          value={item.readonlyText || ''}
+          onChange={(e) => {
+            onChangeProperty('readonlyText', e.target.value);
+          }}
+          helperText="Label to display when the item is shown in a read-only form. If not specified, the item's normal label text is shown."
+        />
+        <LabeledCheckbox
+          label='Read-only'
+          checked={item.readOnly}
+          onChange={(e) =>
+            onChangeProperty('readOnly', (e.target as HTMLInputElement).checked)
+          }
+        />
+        <LabeledCheckbox
+          label='Hidden'
+          checked={item.hidden}
+          onChange={(e) =>
+            onChangeProperty('hidden', (e.target as HTMLInputElement).checked)
+          }
+        />
+        <TextInput
+          // TODO(#5776)
+          label='Mapping for custom field key'
+          value={item.mapping?.customFieldKey || ''}
+          onChange={(e) => {
+            onChangeProperty('mapping', { customFieldKey: e.target.value });
+          }}
+        />
+        <LabeledCheckbox
+          label='Warn if empty'
+          checked={item.warnIfEmpty}
+          onChange={(e) =>
+            onChangeProperty(
+              'warnIfEmpty',
+              (e.target as HTMLInputElement).checked
+            )
+          }
+        />
+        <FormSelect
+          label='Data collected about'
+          value={
+            dataCollectedAboutPickList.find(
+              (o) => o.code === item.dataCollectedAbout
+            ) || undefined
+          }
+          options={dataCollectedAboutPickList}
+          onChange={(_e, value) => {
+            console.log(value);
+            onChangeProperty(
+              'dataCollectedAbout',
+              isPickListOption(value) ? value.code : undefined
+            );
+          }}
+        />
+        <FormSelect
+          label='Size'
+          value={
+            inputSizePickList.find((o) => o.code === item.size) || undefined
+          }
+          options={inputSizePickList}
+          onChange={(_e, value) => {
+            console.log(value);
+            onChangeProperty(
+              'size',
+              isPickListOption(value) ? value.code : undefined
+            );
+          }}
+        />
+        <FormSelect
+          label='Disabled Display'
+          value={
+            disabledDisplayPickList.find(
+              (o) => o.code === item.disabledDisplay
+            ) || undefined
+          }
+          options={disabledDisplayPickList}
+          onChange={(_e, value) => {
+            console.log(value);
+            onChangeProperty(
+              'disabledDisplay',
+              isPickListOption(value) ? value.code : undefined
+            );
+          }}
+        />
+        <TextInput
```

Comment:
This can be a single-select for the [pick list types](https://github.com/greenriver/hmis-frontend/blob/2d45a5c5ec38a020d878b1acfa11e1cdf6dc89d0/src/types/gqlEnums.ts#L913). We can do the whole list for now, though eventually it might make sense to slim it down and/or adjust the available options based on the form type (or other contextual factors).